includes/webhooks.php

```diff
@@ -50,7 +50,7 @@ function fastwc_woocommerce_webhook_options_save( $webhook_id ) {
  * @return bool
  */
 function fastwc_is_disabled_webhook( $webhook ) {
-	return 'disabled' === $webhook->get_status();
+	return ! empty( $webhook ) && 'disabled' === $webhook->get_status();
```

Comment:
Is there a possibility of not having get_status method in variable webhook?

Reply:
No, the webhook is always a `WC_Webhook`, which will always have the `get_status` method.